### DISCUSSION TITLE
Brilliant Gem but where is documentation regarding Action Cable?

### DISCUSSION THREAD

**Discussion by ali-gsquad** [excid3/noticed#93] [Q&A]
How do I send notifications through Action Cable? Am I missing documentation where it explain how do Action Cable receive the broadcast?

**Comment by excid3** [comment]
Great question! I haven't documented it too thoroughly.

We include and broadcast on the `Noticed::NotificationChannel` which you can find here: https://github.com/excid3/noticed/blob/master/lib/noticed/notification_channel.rb

So your Javascript will need to subscribe to that. By default, we simply send over the params as JSON.

**Reply by excid3** [reply]
I've written some documentation here now: https://github.com/excid3/noticed/blob/master/docs/actioncable.md

**Reply by ali-gsquad** [reply]
Thank you!
